src/tripCard/TripCard.style.tsx

```diff
@@ -53,10 +53,8 @@ export const StyledTripCard = styled(Card)`
 
   & .kirk-tripCard-itinerary {
     min-width: 0; /* reset flex default to allow shrinking below content width */
-  }
-
-  &.kirk-tripCard--with-price .kirk-tripCard-itinerary {
-    padding-right: ${space.m};
+    position: relative;
+    left: -${space.xl};
```

Comment:
As we did for other component, the negative margin is not a nice solution but at least consistent. Once we finish the normalisation, we'll remove all !important stuff and be able to override styles nicer.

Reply:
Could add that in a css comment? It would be nice to explain why we need this.

Reply:
Should we add a comment in the code to easily identify/grep the negative margin we will need to update once all components are normalized?